test/lib/creation.go

```diff
@@ -25,11 +25,70 @@ import (
 
 	bindingsv1alpha1 "knative.dev/eventing-kafka/pkg/apis/bindings/v1alpha1"
 	bindingsv1beta1 "knative.dev/eventing-kafka/pkg/apis/bindings/v1beta1"
+	channelsv1alpha1 "knative.dev/eventing-kafka/pkg/apis/messaging/v1alpha1"
+	channelsv1beta1 "knative.dev/eventing-kafka/pkg/apis/messaging/v1beta1"
 	sourcesv1alpha1 "knative.dev/eventing-kafka/pkg/apis/sources/v1alpha1"
 	sourcesv1beta1 "knative.dev/eventing-kafka/pkg/apis/sources/v1beta1"
 	kafkaclientset "knative.dev/eventing-kafka/pkg/client/clientset/versioned"
 )
 
+func CreateKafkaChannelV1Alpha1OrFail(c *testlib.Client, kafkaChannel *channelsv1alpha1.KafkaChannel) {
+	kafkaChannelClientSet, err := kafkaclientset.NewForConfig(c.Config)
+	if err != nil {
+		c.T.Fatalf("Failed to create v1alpha1 KafkaChannel client: %v", err)
+	}
+
+	kChannels := kafkaChannelClientSet.MessagingV1alpha1().KafkaChannels(c.Namespace)
+	if createdKafkaChannel, err := kChannels.Create(context.Background(), kafkaChannel, metav1.CreateOptions{}); err != nil {
+		c.T.Fatalf("Failed to create v1alpha1 KafkaChannel %q: %v", kafkaChannel.Name, err)
+	} else {
+		c.Tracker.AddObj(createdKafkaChannel)
+	}
+}
+
+func GetKafkaChannelV1Alpha1OrFail(c *testlib.Client, kafkaChannel string) *channelsv1alpha1.KafkaChannel {
```

Comment:
Do we need to tes v1alpha1? We're going to remove this code in a week when the next release is out :)

Reply:
I didn't think so, but I also assumed if I **didn't** add it, somebody would say I **should** add it. so, :shrug:

Reply:
For me you should remove them :smile:

Reply:
ok, removed them